apps/webapp/app/send/send-form.tsx

```diff
@@ -13,7 +13,7 @@ const InputToken = dynamic(() => import('../../shared/input-token'), {
   ssr: false,
 });
 
-export default function SendForm() {
+const Send = () => {
```

Comment:
Given this is dynamically imported, don't think you need to dynamically import `InputToken` above right?

Reply:
Yes